contracts/swaggerV1Compat.yml

```diff
@@ -119,7 +119,7 @@ paths:
           name: Accept
           schema:
             type: string
-            description: Specifies how query results should be encoded in the response.
+            description: Specifies how query results should be encoded in the response. **Note:** When using `application/csv`, query results include epoch timestamps instead of RFC3339 timestamps.
```

Comment:
```suggestion
            description: Specifies how query results should be encoded in the response. **Note:** With `application/csv`, query results include epoch timestamps instead of RFC3339 timestamps.
```